pkg/network/common/common.go

```diff
@@ -124,7 +124,7 @@ func (ni *NetworkInfo) CheckClusterObjects(subnets []networkapi.HostSubnet, pods
 		}
 	}
 	for _, pod := range pods {
-		if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.HostNetwork {
+		if pod.Spec.SecurityContext != nil && pod.Spec.HostNetwork {
```

Comment:
Presumably the `SecurityContext != nil` check is no longer needed if `HostNetwork` is no longer a subfield of `SecurityContext`?

Reply:
> Presumably the SecurityContext != nil check is no longer needed if HostNetwork is no longer a subfield of SecurityContext?

fixed.